src/coreclr/tests/src/Interop/PInvoke/Generics/CMakeLists.txt

```diff
@@ -13,6 +13,10 @@ elseif(CLR_CMAKE_TARGET_ARCH_ARM64)
   add_definitions(-DTARGET_ARM64)
   add_definitions(-DTARGET_ARMARCH)
 endif()
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  # The ABI for passing parameters with 32-byte alignment has changed in GCC 4.6
+  add_compile_options(-Wno-psabi)
```

Comment:
@jkotas, just wondering if warning as error state for PR runs is same as before (`ON`), why was this special case required for GCC as master was ok without it?

Reply:
The PR enabled warnings-as-errors for tests. The CoreCLR tests were built without `-Werror` before this change.

Reply:
Thanks, makes sense.